tests/run-pass/hashmap.rs

```diff
@@ -22,4 +23,13 @@ fn main() {
     assert_eq!(map.values().fold(0, |x, y| x+y), num*(num-1)/2);
 
     // TODO: Test Entry API, Iterators, ...
+
+}
+
+fn main() {
+    let map : HashMap<i32, i32, BuildHasherDefault<collections::hash_map::DefaultHasher>> = Default::default();
+    let map_normal: HashMap<i32, i32> = HashMap::new();
+
+    test_map(map);
+    test_map(map_normal);
```

Comment:
This is already a slow test, I do not think we should run the entire test twice.

Running it just with the default hasher seems reasonable.

Reply:
I've changed it to create both maps, but only test the one with the default hasher.

Reply:
I've changed it to create both maps, but only test the one with the default hasher.